### PR TITLE
Feature: annotation view animations

### PR DESCRIPTION
**Pull request by vadymmarkov**
It's nice to animate annotation views when they appear on the map. So here we have 2 animations types (`bounce` and `fade`) that could be applied like this:

```swift
func mapView(_ mapView: MKMapView, didAdd views: [MKAnnotationView]) {
  clusteringManager.animate(annotationViews: views, animation: .bounce)
}
```

### CODE REVIEW

File: Sources/AnnotationAnimation.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    case .fade:
+      return FadeAnnotationAnimator()
+    }
+  }
+}
+
+// MARK: - Protocol
+
+protocol AnnotationAnimator {
+  func animate(views: [MKAnnotationView])
```
Comment (onmyway133) [inline Sources/AnnotationAnimation.swift:21]: Actually I think 1 method with array parameter is enough.

**Review by onmyway133 — APPROVED** [review]

File: Sources/AnnotationAnimation.swift (hunk 1 above)
Comment (vadymmarkov) [reply Sources/AnnotationAnimation.swift:21]: @onmyway133 What do you mean?

Comment (onmyway133) [reply Sources/AnnotationAnimation.swift:21]: Like for 1 item, I can just use `animate(views: [just1View])`

Comment (vadymmarkov) [reply Sources/AnnotationAnimation.swift:21]: Yes, but I didn't want to iterate views in implementation.

**Review by zenangst — APPROVED** [review]
Looks good!

**Pull request merged**